docs/sources/alerting/fundamentals/_index.md

```diff
@@ -91,40 +95,40 @@ A rule using the PromQL expression above creates as many alert instances as the
+### Notification policies
 
-{{< figure src="/media/docs/alerting/notification-routing.png" max-width="750px" caption="Notification policy routing" >}}
+[Notification policies](ref:notification-policies) provide a flexible method to handle alert notifications by routing alerts to contact points via label matching.
 
-### Contact points
+Each notification policy consists of a set of label matchers (0 or more) that specify which alert instances (identified by their labels) they handle. Notification policies are defined as a tree structure, where the root of the notification policy tree is the **Default notification policy**, which ensures all alerts are handled.
 
-[Contact points](ref:contact-points) determine where notifications are sent. For example, you might have a contact point that sends notifications to an email address, to Slack, to an incident management system (IRM) such as Grafana OnCall or Pagerduty, or to a webhook.
+{{< figure src="/media/docs/alerting/notification-routing.png" max-width="750px" caption="Notification policy routing" >}}
+
+Each notification policy is responsible for deciding the contact point and when to send the notification. Additionally, it can [group multiple firing alert instances into a single notification](ref:group-alert-notifications) to reduce alert noise.
 
-Notifications sent from contact points are customizable with notification templates, which can be shared between contact points.
+{{< figure src="/media/docs/alerting/alerting-notification-policy-diagram-v5.png" max-width="750px" alt="A diagram of the notification policy component" >}}
 
 ### Silences and mute timings
 
 [Silences](ref:silences) and [mute timings](ref:mute-timings) allow you to pause notifications for specific alerts or even entire notification policies. Use a silence to pause notifications on an ad-hoc basis, such as during a maintenance window; and use mute timings to pause notifications at regular intervals, such as evenings and weekends.
 
 ### Architecture
 
-Grafana Alerting is built on the Prometheus model of designing alerting systems.
-
-Prometheus-based alerting systems have two main components:
+Grafana Alerting is built on the Prometheus model of designing alerting systems. Prometheus-based alerting systems have two main components:
 
 - An alert generator that evaluates alert rules and sends firing and resolved alerts to the alert receiver.
 - An alert receiver (also known as Alertmanager) that receives the alerts and is responsible for handling them and sending their notifications.
 
-Grafana doesn’t use Prometheus as its default alert generator because Grafana Alerting needs to work with many other data sources in addition to Prometheus.
+Grafana Alerting doesn’t use Prometheus as its default alert generator because it needs to work with many other data sources, not just Prometheus.
```

Comment:
```suggestion
Grafana Alerting doesn’t use Prometheus as its default alert generator because it works with many other data sources; not just Prometheus.
```